.github/workflows/build-and-deploy-dev.yml

```diff
@@ -0,0 +1,131 @@
+name: "DEV: Build and deploy smoke tests"
+env:
+  BUILD_ROLE: arn:aws:iam::706615647326:role/deployers/dev-github-actions-publish-to-s3-for-code-signing
+  DEPLOY_ROLE: arn:aws:iam::653994557586:role/dev-auth-deploy-pipeline-GitHubActionsRole-QrtGginNnjDD
+  SOURCE_BUCKET: di-auth-lambda-source-20220215165054622300000001
+  DESTINATION_BUCKET: di-auth-lambda-signed-20220215165009815000000001
+  SIGNING_PROFILE: di_auth_lambda_signing_20220214175605677200000001
+  ARTIFACT_BUCKET: dev-auth-deploy-pipeline-githubartifactsourcebuck-ssdefc91xjh6
```

Comment:
Do these need to go in GitHub project level environment variables? (I know they are not secret)

Reply:
alas, they can't - they're different in dev / prod and GHA secrets are difficult. there is a refactor coming, once I work out how best to do it!